backend/app/assets/javascripts/spree/backend/components/admin_nav.js

```diff
@@ -1,5 +1,5 @@
 Spree.ready(function() {
-  if (window.screen.width <= 1024 && !document.cookie.includes("admin_nav_hidden")) {
+  if (window.width <= 1024 && !document.cookie.includes("admin_nav_hidden")) {
```

Comment:
Thanks!

Btw. [the recommended way is to use innerWidth](https://developer.mozilla.org/en-US/docs/Web/API/Window/innerWidth) 
```suggestion
  if (window.innerWidth <= 1024 && !document.cookie.includes("admin_nav_hidden")) {
```

Reply:
doesnt matter for this fix